index.bs

```diff
@@ -304,22 +304,42 @@ it might still be possible; see [[#removing-features]].
 
 Do not assume that a change or removal is impossible without first checking.
 
-<h3 id=removing-features>Remove or change capabilities only once you understand existing usage</h3>
-
-Prioritize compatibility with existing content when removing or changing functionality.
-
-Once a significant amount of content has come to depend on a particular behavior,
-removing or changing that behavior is discouraged.
-Removing or changing features and capabilities is possible,
-but it first requires that the nature and scope of the impact on existing content
-is well understood.
-This might require research into how features are used by existing content.
-
-The obligation to understand existing usage also applies to any features that content relies upon.
+<h3 id=removing-features>Prioritize compatibility when changing or removing features</h3>
+
+Before changing how a feature behaves,
+understand how websites are currently using it.
+Gaining a good understanding might require research,
+for example by adding metrics to a popular user agent
+or by [searching the HTTP Archive](https://har.fyi/guides/getting-started/).
+Breaking content harms users,
+and the benefit of a change has to
+significantly outweigh that harm to be worth doing.
+
+The obligation to understand existing usage
+also applies to non-standardized or unspecified features
+that content relies upon.
 This includes vendor-proprietary features and
-behavior that might be considered implementation bugs.
-Web features are not solely defined in specifications;
-they are also defined by how content uses those features.
+implementation bugs.
+
+Despite this, it is sometimes acceptable
+to break some existing content
+in order to help a large number of the web's users.
+Breakage is more likely to be acceptable if:
+
+* Only a tiny amount of existing content depend on the feature or behavior.
+* Only a tiny number of people see that content.
+* The content only appears in test cases or examples.
+* The content is currently broken in most recent popular user agents.
```

Comment:
Does it matter if the user agent is popular if the change already affects most user agents?

Reply:
I added more detail about why it matters that the other UAs are popular: it makes it more likely that they would have received bug reports if anyone's relying on the content, and it means that if we're increasing interoperability, we're not breaking the dramatically larger number of users.